cves/CVE-2009-1891.yml

```diff
@@ -97,8 +99,8 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
-  date:
+  answer: The vulnerablility was found by an independent user, whom submitted a bug report.
+  date: 2009-06-26
   automated:
```

Comment:
If the vulnerability was found by a user then automated, google, and contest should be false (true for contest if there was an actual contest for this)